crm_secondary_salesperson/models/crm_lead.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 - TODAY, Marcel Savegnago <marcel.savegnago@escodoo.com.br> - Escodoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class CrmLead(models.Model):
+
+    _inherit = 'crm.lead'
+
+    secondary_user_id = fields.Many2one(
```

Comment:
Maybe you could add a constraint: user_id <> sesconday_user_id ??????

Reply:
Done